docs/data_quality.md

```diff
@@ -0,0 +1,111 @@
+# Properties of a High Quality OSV Record
+
+## Version
+
+1.0.0 (SEMVER)
+
+## Purpose
+
+Describe the “good enough” OSV record that will be imported by OSV.dev
+
+### Out of scope
+
+This does not discuss the problem of record bit rot over time, after initial successful import. The problem of continuous revalidation and treatment of records that have been successfully imported will be dealt with separately in the companion to this, *Managing the Perishability of OSV Records*.
+
+Deferred to a future iteration: validating the existence of vulnerable functions in the `ecosystem_specific` field, if supplied.
+
+## Audience
+
+1. OSV record producers
+2. Downstream OSV.dev record consumers
+
+## Rationale
+
+OSV.dev seeks to be a comprehensive, accurate and timely database of known vulnerabilities that is highly automation friendly. In order to meet this accuracy goal, a quality bar needs to be both defined and sustainably enforced.
```

Comment:
Just out of curiosity given that it can't be sustainably enforced for OSV records coming from OSS-Fuzz (https://github.com/google/osv.dev/issues/2176) I wonder if they will no longer be imported (automatically)? If so I can probably close https://github.com/google/osv.dev/pull/2222.

Reply:
Got it. As far as I understand it should be addressed in https://github.com/google/osv.dev/issues/2188 one way or another. Until then it would maybe make sense to mention that the quality bar isn't enforced yet. It should hopefully help to prevent treating all OSV entries as vulnerabilities. I'm 99% sure that bogus CVEs based on OSV data popped up because the OSV database was (and still is) advertised in a way that doesn't (always) match reality.

Reply:
I think this discussion is separate to the intention of this document here. 

This document is meant to serve as a reference for how to properly structure and format an OSV record. It doesn't go into whether or not a record is worthy of being considered a vulnerability. That would be a separate discussion :)

Reply:
> It doesn't go into whether or not a record is worthy of being considered a vulnerability

Currently the documentation says "It is also assumed that the vulnerability discussed in the OSV record is valid and affects the software described" and looking at https://github.com/google/osv.dev/pull/2193#discussion_r1621508490 I think it implies that OSV records are vulnerabilities.

Either way I agree that this discussion can be moved elsewhere. Just to clarify I think OSS-Fuzz is great and OSV is great. It's just that they don't interact well with each other yet and get misused from time to time by some organizations.